README.md

```diff
@@ -12,19 +12,173 @@ JavaScript runtimes:
 
 Instead of using FFI and the libextism shared object, this library uses whatever Wasm runtime is already available with the JavaScript runtime.
 
+## Installation
 
-## Install
-
+Install via npm:
 ```
 npm install @extism/extism@1.0.0-rc1 --save
 ```
 
 > **Note**: Keep in mind we will possibly have breaking changes b/w rc versions until we hit 1.0.
 
-## API
+## Getting Started
+
+This guide should walk you through some of the concepts in Extism and this JS library.
+
+First you should import `createPlugin` and `ExtismPluginOptions` from Extism:
+```js
+// CommonJS
+const createPlugin = require("../dist/node/index")
```

Comment:
Can we write this from the user's perspective? how do we import from the npm module? If i'm using typescript how do i get the types I need?